### PR TITLE
Fixes #79

### LINKED ISSUE #79: Allow the height of the shadow to be changed
Users may want to customize the height of the shadow, so some of these properties can be moved before the `@apply`.

``` css
  #dropShadow {
        @apply(--paper-header-panel-shadow);
        position: absolute;
        top: 0;
        left: 0;
        right: 0;
        height: 6px;
        pointer-events: none;
        -webkit-transition: opacity 0.5s;
        transition: opacity 0.5s;
        opacity: 0;
      }
```

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: paper-header-panel.html [hunk 1]
```diff
@@ -232,19 +227,20 @@
       }
 
       #dropShadow {
+        -webkit-transition: opacity 0.5s;
```
Comment (frankiefu) [inline paper-header-panel.html]: Remove the prefix one since it's not needed.

File: paper-header-panel.html [hunk 2]
```diff
@@ -232,19 +227,20 @@
...
+        transition: opacity 0.5s;
+        height: 6px;
+        bottom: -6px;
```
Comment (frankiefu) [inline paper-header-panel.html]: Why do we need `bottom`?  Seem like we don't need it.

**Comment by frankiefu** [comment]
It's not related to this fix.  But it would be good to fix the demo so it doesn't complain about `/deep/ combinator is deprecated`

**Comment by blasten** [comment]
> It's not related to this fix. But it would be good to fix the demo so it doesn't complain about /deep/ combinator is deprecated

@frankiefu That's fixed :)

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**